control-plane/helper/test/test_util.go

```diff
@@ -43,6 +44,8 @@ type TestServerClient struct {
 	Watcher    consul.ServerConnectionManager
 }
 
+var TestPrometheusSink = &prometheus.PrometheusSink{}
```

Comment:
why do we need this global here? can we create this in the `testSink` function and just use it there?